tests/hazmat/primitives/test_scrypt.py

```diff
@@ -77,6 +97,7 @@ def test_password_not_bytes(self, backend):
 
     @pytest.mark.parametrize("params", vectors)
     def test_verify(self, backend, params):
+        _skip_if_memory_limited(backend._scrypt_mem_limit, params)
```

Comment:
is there a reason not to just write `sys.maxsize // 2` here instead of the backend dance?